script/Arcueid/Day 5/05_00_ARC05_6_2.txt

```diff
@@ -552,7 +558,7 @@ A river of flame appears once again at Vlov's feet.
 [sha:43ed0cfcecec28763cda4ee09c52a75a816f94d2]{
 -- Page 500, Offset 9932.
 -- 生命のみを燃やす火に包まれ、緑の蔦は灰に<還|かえ>った。
-Engulfed by a fire that burns only life, the green ivy is reduced to ashes.
+Engulfed by a fire that burns only life, the green ivy is reduced to ash.
```

Comment:
Plural reads better to me (also more ngrammed)

Reply:
I don't really give much credence to the ngram because its singular vs. plural. imo, "ash" is correct because "green ivy" is referring to the ivy as a whole, it's singular in that way grammatically.